internal/downloader/downloader.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//////////////////////////////////////////////////////////////////////////////
+//
+// Provide a thin-as-possible wrapper for conftest's `Download` function,
+// which is itself mostly a wrapper for hashicorp/go-getter.
+//
+// Conftest's download function already has support for a lot of different
+// sources that we might need in future, so let's use it rather than do our
+// own fetching.
+//
+//
+package downloader
+
+import (
+	"context"
+
+	conftestDownloader "github.com/open-policy-agent/conftest/downloader"
```

Comment:
Nitpick, we usually don't see package names this long, so perhaps
```suggestion
	d "github.com/open-policy-agent/conftest/downloader"
```

or

```suggestion
	conftest "github.com/open-policy-agent/conftest/downloader"
```

Reply:
Yes, we must avoid readable code at all costs when using golang. :stuck_out_tongue_winking_eye:

Reply:
![image](https://user-images.githubusercontent.com/1306050/191780915-91995526-e3e1-442f-9dc9-5219211bbb23.png)